### DISCUSSION TITLE
Face cloaking

### DISCUSSION THREAD

**Discussion by make-github-pseudonymous-again** [photoprism/photoprism#1886] [Ideas]
Is there a use-case for [Fawkes](https://sandlab.cs.uchicago.edu/fawkes)-processed downloads? There could be a download button that says: "download a version of the image that has been made more resistant to face recognition algorithms". Although much more complicated to implement than metadata-stripping, it could be part of a more general "download a version of the image for public use".

**Comment by graciousgrey** [comment]
You are the first one asking for it ;)
